packages/yew/src/dom_bundle/blist.rs

```diff
@@ -453,6 +453,47 @@ impl Reconcilable for VList {
     }
 }
 
+#[cfg(feature = "hydration")]
+mod feat_hydration {
+    use super::*;
+
+    use crate::dom_bundle::{Fragment, Hydratable};
+
+    impl Hydratable for VList {
+        fn hydrate(
+            self,
+            root: &BSubtree,
+            parent_scope: &AnyScope,
+            parent: &Element,
+            fragment: &mut Fragment,
+        ) -> (NodeRef, Self::Bundle) {
+            let node_ref = NodeRef::default();
+            let mut children = Vec::with_capacity(self.children.len());
+
+            for (index, child) in self.children.into_iter().enumerate() {
+                let (child_node_ref, child) = child.hydrate(root, parent_scope, parent, fragment);
+
+                if index == 0 {
+                    node_ref.reuse(child_node_ref);
+                }
```

Comment:
No, it triggers a follow up render to fix all `NodeRef`s.

There's no guarantee a correct node will be linked at this moment as it may be a comment node that is going to be removed.